src/translate.rs

```diff
@@ -1162,38 +1176,67 @@ impl FromGlib<i32> for Option<u64> {
     }
 }
 
-/// Translate from a pointer type without taking ownership, transfer: none.
+/// Translate from a pointer type which is annotated with `transfer: none`.
+/// The resulting value is referenced at least once, by the bindings.
+/// Values obtained using this trait must be unreferenced on `drop()`
+/// by the implementing type.
+///
+/// This is suitable for floating references, which become strong references.
```

Comment:
Maybe with a link to some wider explanation about floating references, see above